satpy/readers/olci_nc.py

```diff
@@ -166,10 +176,12 @@ def __init__(self, filename, filename_info, filetype_info, engine=None):
 class NCOLCI1B(NCOLCIChannelBase):
     """File handler for OLCI l1b."""
 
-    def __init__(self, filename, filename_info, filetype_info, cal, engine=None):
+    def __init__(self, filename, filename_info, filetype_info, cal=None, engine=None, mask_items=None):
         """Init the file handler."""
         super().__init__(filename, filename_info, filetype_info, engine)
-        self.cal = cal.nc
+        if cal is not None:
```

Comment:
In what conditions is `cal` None? Should this be required (but keep it as a kwarg for consistency) and raise an error if it is None?

Reply:
The `cal` will be `None` when loading `mask` into the scene. In that case, that is alright since we only care about the `mask_items`.